pkg/rewards/flags.go

```diff
@@ -91,4 +91,11 @@ var (
 		Value:   "all",
 		EnvVars: []string{"REWARDS_CLAIM_TYPE"},
 	}
+
+	SilentFlag = cli.BoolFlag{
+		Name:    "silent",
+		Aliases: []string{"s"},
+		Usage:   "Suppress output except for claim",
+		EnvVars: []string{"REWARDS_CLAIM_SILENT"},
```

Comment:
this should be put in common flags since it will eventually be used in other commands

Reply:
Did you forget to remove it or my mobile view or wrong?